### ISSUE TITLE
Bling uses deprecated method for accessing PropTypes

### ISSUE THREAD

**Issue by capndesign** [nfl/react-gpt#25]
I just upgraded to React 15.5.x and got the following warning in my console (with a partial stack trace):

```
warning.js:36 Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
printWarning @ warning.js:36
warning @ warning.js:60
get @ React.js:95
(anonymous) @ Bling.js:664
...
```

It appears to be pointing to [this line](https://github.com/nfl/react-gpt/blob/master/src/Bling.js#L29). Also, here are the [React 15.5.0 release notes](https://github.com/facebook/react/releases/tag/v15.5.0).

**Comment by jamsea** [comment]
@capndesign it looks like the fix is to `npm install --save-dev prop-types` and change the import statement. We haven't upgraded React to it's latest version yet so this isn't high priority, but if you send us a PR we'll gladly merge it in!

**Comment by nhducit** [comment]
@jamsea I created a PR. Please help to review it. thanks

 https://github.com/nfl/react-gpt/pull/32

**Comment by potench** [comment]
Fixed in #32 #34
